app/src/main/java/ru/ifmo/md/extratask1/SquareImageView.java

```diff
@@ -0,0 +1,25 @@
+package ru.ifmo.md.extratask1;
```

Comment:
You said in several pull requests that this class is reduntant. However, you didn't say anything about it in our [https://github.com/IFMO-MobDev-2014/lesson3/pull/2](lesson3), so I figured it's okay to use this class.